index.bs

```diff
@@ -12188,18 +12190,19 @@ internal method as follows.
 
 <h4 id="legacy-platform-object-abstract-ops">Abstract operations</h4>
 
-<div algorithm>
+<div algorithm="to determine if a property name is an array index">
 
-    The name of each property that appears to exist due to an object supporting indexed properties
-    is an <dfn id="dfn-array-index-property-name" export>array index property name</dfn>,
-    which is a property name |P| for which the following algorithm returns true:
+    To determine if a property name |P| <dfn lt="is an array index|is not an array index"
+    oldids="dfn-array-index-property-name">is an [=array index=]</dfn>, the following algorithm is
```

Comment:
I'm not too sure I like the name change. Aren't array indices numbers? And aren't we here verifying they're in fact a string? While more verbose, the old name made that a lot more obvious.

Reply:
Not really, array indices are strings according to [ECMAScript](https://tc39.github.io/ecma262/#array-index).

While the old name *is* more obvious, I'm explicitly moving the *definition* of that term out of IDL, as I do not want to provide a competing definition of what's essencially the same thing under a different name. Instead IDL should only offering an algorithm to verify that an ECMAScript value is indeed an array index, as ES does not provide that, not in algorithmic form at least.

Reply:
> Not really, array indices are strings according to ECMAScript.

Well that settles it, then. You can just ignore my comment. :)